src/test/java/io/confluent/idesidecar/restapi/util/CPDemoTestEnvironment.java

```diff
@@ -149,7 +157,7 @@ private void setMinISR() {
     try {
       kafka1.execInContainer(
           "kafka-configs",
-          "--bootstrap-server", "kafka1:12091",
+          "--bootstrap-server", "kafka1:12093",
```

Comment:
I think `kafka1:12091` didn't work before inside the Docker network because this port is assigned to the `CLEAR` listener that is using `localhost` as the advertised listener.